### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./options/locale,./options/license,./public/vendor -L actived,allways,attachements,ba,befores,commiter,pullrequest,pullrequests,readby,splitted,te,unknwon`

### CODE REVIEW

File: docs/content/doc/advanced/config-cheat-sheet.zh-cn.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ ALLOW_DATA_URI_IMAGES = true
 - `ALLOW_DATA_URI_IMAGES`: **false** 允许 data uri 图片 (`<img src="data:image/png;base64,..."/>`)。
 
 多个净化规则可以被同时定义，只要section名称最后一位不重复即可。如： `[markup.sanitizer.TeX-2]`。
-为了针对一种渲染类型进行一个特殊的净化策略，必须使用形如 `[markup.sanitizer.asciidoc.rule-1]` 的方式来命名 seciton。
+为了针对一种渲染类型进行一个特殊的净化策略，必须使用形如 `[markup.sanitizer.asciidoc.rule-1]` 的方式来命名 section。
```
Comment (luzpaz) [inline docs/content/doc/advanced/config-cheat-sheet.zh-cn.md:349]: The english word in this translated file is incorrect, therefore I fixed it.

File: web_src/fomantic/build/semantic.css [hunk 2]
```diff
@@ -30873,7 +30873,7 @@ ol.ui.suffixed.list li:before,
      List
 ---------------*/
 
-/* Menu divider shouldnt apply */
```
Comment (6543) [inline web_src/fomantic/build/semantic.css]: if i remember this is generated stuff?

Comment (luzpaz) [reply web_src/fomantic/build/semantic.css]: Where from ?

Edit: https://github.com/fomantic/Fomantic-UI ?

Comment (zeripath) [reply web_src/fomantic/build/semantic.css]: You should revert the changes to web_src/fomantic/build/semantic.css and web_src/fomantic/build/semantic.js as these are both generated.

Comment (luzpaz) [reply web_src/fomantic/build/semantic.css]: Done. 13ae824

File: Makefile [hunk 3]
```diff
@@ -771,7 +771,7 @@ generate-manpage:
 	@mkdir -p man/man1/ man/man5
 	@./gitea docs --man > man/man1/gitea.1
 	@gzip -9 man/man1/gitea.1 && echo man/man1/gitea.1.gz created
-	@#TODO A smal script witch format config-cheat-sheet.en-us.md nicely to suit as config man page
+	@#TODO A small script which formats config-cheat-sheet.en-us.md nicely to suit as config man page
```
Comment (zeripath) [inline Makefile]:
```suggestion
	@#TODO A small script that formats config-cheat-sheet.en-us.md nicely for use as a config man page
```

**Review by zeripath — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**